pkgs/by-name/bi/bitwarden-desktop/package.nix

```diff
@@ -173,6 +181,10 @@ in buildNpmPackage rec {
       --set-default ELECTRON_IS_DEV 0 \
       --inherit-argv0
 
+    # The polkit policy file should be kept up to date with the one bitwarden-desktop source code; see
+    # https://github.com/bitwarden/clients/blob/76021b48171b5471a4872cb9d9404c50fbc2fe05/apps/desktop/src/platform/main/biometric/biometric.unix.main.ts#L12-L27
+    install -Dm644 ${./com.bitwarden.Bitwarden.policy} $out/share/polkit-1/actions/com.bitwarden.Bitwarden.policy
```

Comment:
Thoughts: we can probably extract the content and filename directly from original typescript file, e.g. `ts-node -e "import './biometric.unix.main.ts'; console.log(polkitPolicy);"`